cypress/integration/search.spec.js

```diff
@@ -30,6 +30,40 @@ context('Searching', () => {
         .and('contain', 'wâpamêw') // lemma
         .and('contain', 'Action word') // verb
     })
+
+    it('should show linguistic breakdowns as an ordered list when the user clicks on the ? icon beside a word', () => {
+      // begin from the homepage
+      cy.visit('/');
+
+      // lock onto the searchbar
+      cy.get('[data-cy=search]')
+      // get a word (nipaw)
+      .type('nipaw');
+      
+      // tab through the elements to force the tooltip to pop up
+      cy.get('[data-cy=information-mark]').first().click({force:true});
+
+      // see the linguistic breakdown as an ordered list
+      cy.get('[data-cy=linguistic-breakdown]').children().eq(1).should('have', 'li');
+      
+    })
+
+    it('should allow the tooltip to be focused on when the user tabs through it', () => {
+      // goodness, that's a mouthful and should _probably_ be worded better.
+      // begin from the homepage
+      cy.visit('/');
+      // lock onto the searchbar
+      cy.get('[data-cy=search]')
+      // get a word (use nipaw)
+      .type('nipaw');
+
+      // tab through the page elements until arriving on the '?' icon
+      cy.get('[data-cy=information-mark]').first().click({force:true});
+      
+      // it should trigger the focus icon's outline's focused state
+      cy.get('[data-cy=information-mark]').focus().should('have.css', 'outline', 'rgb(28, 157, 254) solid 3px'); // apparently, Cypress likes for the colours to be in the style of rgb() so we can't default to the CSS variables we've been using. Oof.
+    })
```

Comment:
This is a weird and fragile test, and will break if we decide to change some colours later. Is there a way to check for the outline without checking its colour? If not, we can keep this test as is. It will just be annoying if we ever decide to tweak colours. ¯\\\_(ツ)\_/¯

Reply:
Good eye! I just tested it as `.should('have.css', 'outline')` and that actually works well without testing for the colour, so I'll commit that 💪